pkg/api/sharedresource/v1alpha1/share_configmap_type.go

```diff
@@ -0,0 +1,73 @@
+package v1alpha1
```

Comment:
Should this file be named `shared_configmap_type.go` ?

Reply:
yeah and the secret one too